### PR TITLE
Add CXXFLAGS to compilation

### PR DESCRIPTION
**Pull request by david-waterworth**
I hereby agree to the terms of the CLA available at: [link](https://yandex.ru/legal/cla/?lang=en)

ya.conf sets CATBOOST_OPENSOURCE in CFLAGS but not CXXFLAGS. On my machine at least (Ubuntu 18.04) this results in the GPU trainer not being included in the final build despite the CUDA files compiling.

### CODE REVIEW

File: ya.conf [hunk 1]
```diff
@@ -8,6 +8,7 @@ USE_ASMLIB="no"
 USE_ARCADIA_CUDA="no"
 CATBOOST_OPENSOURCE="yes"
 CFLAGS="-DCATBOOST_OPENSOURCE=yes"
+CXXSFLAGS="-DCATBOOST_OPENSOURCE=yes"
```
Comment (exprmntr) [inline ya.conf]: Is it a typo (CXX**S**FLAGS)?

**Comment by david-waterworth** [comment]
Yes sorry in the progress of creating a clean branch for the pull request I messed up the spelling. I’ll update in the weekend

File: ya.conf (hunk 1 above)
Comment (david-waterworth) [reply ya.conf]: Yes sorry in the progress of creating a clean branch for the pull request I messed up the spelling. I’ll update in the weekend

Comment (david-waterworth) [reply ya.conf]: I've pushed a correction

**Comment by annaveronika** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
CLA already signed

**Pull request merged**